pkg/upgrade/main.go

```diff
@@ -0,0 +1,36 @@
+package upgrade
+
+import (
+	"sort"
+
+	"github.com/Masterminds/semver"
+)
+
+func init() {
+	parseSemVer()
+}
+
+func parseSemVer() {
+	// ignore errors, we shouldn't have semantic version parsing errors at runtime
+	semanticVersions, _ = versions(upgrades)
+	startUpdatesVersion, _ = semver.NewVersion("1.11.0")
+}
+
+// Versions return the list of semantic version sorted
+func versions(versions map[string]upgradeFunction) ([]*semver.Version, error) {
```

Comment:
Could you create a test for this method to verify it returns a correctly sorted list of semver versions?

Reply:
Yes. Done